core/domain/topic_services_test.py

```diff
@@ -147,6 +147,29 @@ def test_update_topic(self):
         self.assertEqual(topic_summary.version, 3)
 
 
+    def test_delete_skill(self):
+        topic_services.delete_skill(
+            self.user_id_admin, self.TOPIC_ID, self.skill_id)
+        topic = topic_services.get_topic_by_id(self.TOPIC_ID)
+        self.assertEqual(topic.skill_ids, [])
+
+
+    def test_delete_story(self):
```

Comment:
Separate non-top-level methods by one line, not two.

/cc @apb7 -- I keep encountering this in reviews, FYI.

Reply:
Done.